eslint.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
         `\n * Copyright The OpenTelemetry Authors\n *\n * Licensed under the Apache License, Version 2.0 (the "License");\n * you may not use this file except in compliance with the License.\n * You may obtain a copy of the License at\n *\n *      https://www.apache.org/licenses/LICENSE-2.0\n *\n * Unless required by applicable law or agreed to in writing, software\n * distributed under the License is distributed on an "AS IS" BASIS,\n * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n * See the License for the specific language governing permissions and\n * limitations under the License.\n `
     }]],
     "import/no-extraneous-dependencies": ["error", { devDependencies: ["test/**/*.ts"] }],
+    "no-process-exit": 0,
```

Comment:
I see it is deprecated for eslint >= 7, so I'm just wondering how it is working :) ?

Reply:
I understand it is working somehow, right? if so, then you can resolve conversation